Pod/Classes/VOKCoreDataManager.m

```diff
@@ -296,6 +323,38 @@ - (NSArray *)importArray:(NSArray *)inputArray forClass:(Class)objectClass withC
     return [returnArray copy];
 }
 
+- (NSArray *)arrayToImportAfterImportingRepeatedUniqueKeys:(NSArray *)repeatedUniqueKeys
+                                   forForeignUniqueKeyPath:(NSString *)foreignUniqueKeyPath
+                                           usingInputArray:(NSArray *)inputArray
+                                                  forClass:(Class)objectClass
+                                               withContext:(NSManagedObjectContext *)contextOrNil
+{
+    NSMutableArray *normalNumberOfKeys = [NSMutableArray arrayWithArray:inputArray];
```

Comment:
I think this variable could have a better name, it's not an NSNumber.  Maybe `uniquedInputArray`?